lib/api/notification.js

```diff
@@ -0,0 +1,18 @@
+const Request = require('./request')
+
+class Notification extends Request{
```

Comment:
Should you define notification types to be used by sdk user?

Reply:
I would leave this part on developers that will use Amio. Let's leave it generic untill we collect more info about how our users use this lib. ;)